src/components/PathwayCourse/index.js

```diff
@@ -466,20 +531,40 @@ function PathwayCourse() {
               </Grid>
             ))}
           </Grid>
-          {/* <Grid  sx={{mb:15}}align="center">
+          <Grid onClick={completedAll ? handleModal : handleSnackbar} sx={{mb:15}}align="center">
             <Grid sx={{mb:3}} >
               <img src={require("./asset/separator.svg")} alt="icon" />
             </Grid>
-            <Grid>
-              <img
-                src={require("./asset/certificate.svg")}
-                alt="certificate icon"
-              />
+            <Grid sx={{cursor: "pointer"}}>
+              {
+                completedAll ? 
+                <CertificateIconColored
+                  className={classes.certificateIcon}
+                /> : 
+                <CertificateIcon
+                  className={classes.certificateIcon}
+                />
+              }
               <Typography sx={{mt:2}} variant="body1" mb={2}>
-                {pathwayCourseData?.pathway} Certificate (Locked)
+                {pathwayCourseData?.pathway} Certificate
               </Typography>
             </Grid>
-          </Grid> */}
+            <Snackbar
+              open={openSnackbar}
+              autoHideDuration={6000}
+              message={`Please complete all the courses to unlock ${pathwayCourseData?.pathway}  certificate`}
+              onClose={handleSnackbar}
+              ContentProps={{
+                sx: {
+                  background: "#2E2E2E",
+                  fontWeight: "400",
+                  fontSize: "18px",
```

Comment:
Using fixed font sizes for fonts is bad for accessibility: What if the user is visually impaired?